package.json

```diff
@@ -148,6 +150,7 @@
     "svg-inline-loader": "^0.8.0",
     "url-loader": "^1.0.1",
     "utf-8-validate": "^5.0.2",
+    "webpack-sources": "1.0.1",
```

Comment:
I think this package isn't used anywhere, could you remove this dependency?

Reply:
yes i will remove this.